test/blueprints/Do.Test.Blueprints.Service/Authentication/Authentication.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Do.Test;
+
+public class Authentication(TimeProvider _timeProvider)
+{
+    public DateTime GetTime() => _timeProvider.GetNow();
```

Comment:
this `GetTime` method has also another meaning, it is a method to test fixed token authentication middleware, so keep a method that does nothing to keep that test as well